test/src/sdk-harness/test_projects/storage_vec/mod.rs

```diff
@@ -0,0 +1,122 @@
+mod utils;
+
+use utils::{
+    setup::get_contract_instance,
+    wrappers::*,
+};
+
+
+mod success {
+    use super::*;
```

Comment:
Except here, this could become a pain but you could import explicitly here and arguably you should (I don't bother)